LibraBFT/ImplShared/Consensus/Types/EpochIndep.agda

```diff
@@ -64,11 +64,25 @@ module LibraBFT.ImplShared.Consensus.Types.EpochIndep where
       _biRound : Round
       _biId    : HashValue
       -- This has more fields...
```

Comment:
I've filled out more of these fields in #74, including fixing the hashing bug and proof about it.  It should not be too painful to merge.

Reply:
no problem